src/main/java/org/folio/rest/impl/LocationUnitApi.java

```diff
@@ -42,7 +44,7 @@ public void getLocationUnitsInstitutions(String query, String totalRecords, int
       cql = StorageHelper.getCql(query, limit, offset, INSTITUTION_TABLE);
     } catch (Exception e) {
       String message = StorageHelper.logAndSaveError(e);
-      asyncResultHandler.handle(Future.succeededFuture(
+      asyncResultHandler.handle(succeededFuture(
```

Comment:
We should probably revert these (static class calls) changes that are not related to Campus, in my opinion